packages/api/src/graphql/mutation.public.ts

```diff
@@ -85,14 +86,17 @@ export const GraphQLPublicMutation = new GraphQLObjectType<undefined, Context>({
 
     // Comment
     // =======
-    createComment: {
+    addPublicComment: {
       type: GraphQLNonNull(GraphQLPublicComment),
       args: {input: {type: GraphQLNonNull(GraphQLCommentInput)}},
-      async resolve(root, {input}, {authenticate, dbAdapter}) {
+      async resolve(_, {input}, {authenticate, dbAdapter}) {
         const {roles} = authenticate()
-        authorise(CanCreateComment, roles)
-        return await dbAdapter.comment.createComment({
-          input: {...input}
+        authorise(CanAddPublicComment, roles)
```

Comment:
Remove this. At the moment everybody that can authenticate is allowed to create comments. And if we change this we will use different permissions for public users.

Reply:
fixed